src/main/java/com/redhat/devtools/intellij/tektoncd/utils/CRDHelper.java

```diff
@@ -20,9 +20,11 @@
     public static CustomResourceDefinitionContext getCRDContext(String apiVersion, String plural) {
         if (Strings.isNullOrEmpty(apiVersion) || Strings.isNullOrEmpty(plural)) return null;
         String[] groupVersion = apiVersion.split("/");
-        if (groupVersion.length != 2) return null;
+        if (groupVersion.length != 2 && !apiVersion.equalsIgnoreCase("triggers.tekton.dev")) {
```

Comment:
I don't understand this test ? Does it mean this code is only relevant for trigger related resources ?

Reply:
Reason in comment below. Let me know what you think